pennylane/qnodes/decorator.py

```diff
@@ -28,7 +28,7 @@
 ALLOWED_INTERFACES = ("autograd", "numpy", "torch", "tf")
 
 
-def QNode(func, device, *, interface="autograd", mutable=True, diff_method="best", properties=None):
+def QNode(func, device, *, interface="autograd", mutable=True, diff_method="best", h=None, properties=None):
```

Comment:
As we add more QNodes, there will likely be a case where the QNodes take different keyword arguments, and it will be unmaintainable to keep adding arguments to the constructor function. Instead, perhaps should do the following:

* The QNode signature accepts `**kwargs` or an `options` dictionary. This dictionary is passed directly to the QNode to parse.

Do people prefer `**kwargs` or `options`? I'm undecided

Reply:
It also avoids us having to update the constructor _and_ the decorator each time 😆

Reply:
It also avoids Sphinx displaying the default value as `None`, which might be confusing to users

Reply:
Sounds good to me! Should that then mean that `properties` would also be passed as `kwargs`/`options` in this case and be passed on to the corresponding `QNode` that is being created?

Reply:
Oh good catch! We already had a options dictionary, it was just called properties 😆 In this case, we should use the existing `properties`, although I'm not sure I like the name.

What do you like in terms of naming? Since I don't think `properties` is being used, now is the perfect time to rename it if we want.

Reply:
I don't mind either :) I'll go with `kwargs` for now, but I'm up for any names that you think could make sense

Reply:
I like `**kwargs`!